docs/concepts/rejection.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Rejection
+headline: Rejection
+bodyclass: docs
+layout: docs
+sidenav: doc-side-concepts-nav.html
+type: markdown
+---
+<h2 class="top">Rejection</h2> 
+
+Rejection is a state of business logic which can be handled by the end user. 
+Consider the following examples of rejections: 
+* “credit card validation declined”, 
```

Comment:
I would put it as type names:
`CreditCardValidationDecllined`,
`OrderCannotBeEmpty`
`InsufficientFunds`.

Also, the last sentence put before this text would make such names understandable.

Reply:
From what I can see in the text, this is not yet resolved.